data/virtualization/autoyast/host_15.xml.ep

```diff
@@ -291,14 +291,54 @@
   </users>
   <scripts>
     <post-scripts config:type="list">
+      % if ($check_var->('SYSTEM_ROLE', 'xen') && $get_var->('VERSION') gt '15-SP5') {
       <script>
-        <filename>debug_libvirtd.sh</filename>
+        <filename>configure_xen_modular_libvirt_log.sh</filename>
         <source><![CDATA[
-cat >> /etc/libvirt/libvirtd.conf<< EOF
+for drv in xen network nodedev nwfilter secret storage
+do
+cat >> "/etc/libvirt/virt${drv}d.conf" << EOF
+log_level = 1
+log_filters="1:qemu 1:libxl 1:libvirt 4:object 4:json 4:event 3:util 1:util.pci"
+log_outputs="1:file:/var/log/libvirt/virt${drv}d.log"
+EOF
+done
+]]>
+        </source>
+      </script>
+      % }
+      % if ($check_var->('SYSTEM_ROLE', 'kvm') && $get_var->('VERSION') gt '15-SP5') {
+      <script>
+        <filename>configure_kvm_modular_libvirt_log.sh</filename>
+        <source><![CDATA[
+for drv in qemu network nodedev nwfilter secret storage
+do
+cat >> "/etc/libvirt/virt${drv}d.conf" << EOF
+log_level = 1
+log_filters="1:qemu 1:libxl 1:libvirt 4:object 4:json 4:event 3:util 1:util.pci"
+log_outputs="1:file:/var/log/libvirt/virt${drv}d.log"
+EOF
+done
+]]>
+        </source>
+      </script>
+      % }
+      % if ($get_var->('VERSION') lt '15-SP6') {
+      <script>
+        <filename>configure_monolithic_libvirt_log.sh</filename>
+        <source><![CDATA[
+cat >> "/etc/libvirt/libvirtd.conf" << EOF
```

Comment:
There is log deamon for kvm. Can not see it.

Reply:
Full daemons are:
```
    local daemons="network nodedev nwfilter secret storage proxy lock "
        
    if (is_kvm_host); then daemons+="qemu log"; else daemons+="xen"; fi

```

Reply:
Added more log for kvm and xen when using modular libvirt.

Reply:
xen: http://10.100.103.207/tests/283/file/autoyast-host_15.xml 
kvm: http://10.100.103.207/tests/284/file/autoyast-host_15.xml

Reply:
I mean daemons specified here are not complete. The full list is:

```
    local daemons="network nodedev nwfilter secret storage proxy lock "
        
    if (is_kvm_host); then daemons+="qemu log"; else daemons+="xen"; fi
```